common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1639,4 +1639,11 @@ public final class AdminConstants {
             "com_zimbra_docs_modern", "com_zimbra_drive_modern", "com_zextras_drive", "com_zextras_drive_open",
             "com_zextras_chat_open", "com_zextras_talk", "zimbra-zimlet-briefcase-edit-lool");
 
+    public static final String CACHE_REFRESHED = "cacheRefreshed";
```

Comment:
this sounds pretty generic for this project. please append license to this constant

Reply:
updated as LICENSE_CACHE_REFRESHED